collectors/python.d.plugin/isc_dhcpd/isc_dhcpd.chart.py

```diff
@@ -86,14 +86,23 @@ class Pool:
     def __init__(self, name, network):
         self.id = re.sub(r'[:/.-]+', '_', name)
         self.name = name
-        self.network = ipaddress.ip_network(address=u'%s' % network)
+
+        networks = network.split(" ")
+        if len(networks) == 1:
+            self.networks = [ ipaddress.ip_network(address=u'%s' % network) ]
+        elif len(networks) == 2:
+            self.networks = list( ipaddress.summarize_address_range(ipaddress.IPv4Address(u'%s' % networks[0]), ipaddress.IPv4Address(u'%s' % networks[1])) )
```

Comment:
Let's see what you think about this implementation.
You just have to define initial and final ip.
range: '192.168.5.99 192.168.5.151'